src/main/java/io/vlingo/xoom/schemata/codegen/TypeDefinitionCompiler.java

```diff
@@ -52,7 +53,7 @@ public static TypeDefinitionCompiler compilerFor(final Stage stage, final String
   public static TypeDefinitionCompiler newCompilerFor(final Stage stage, final String language) {
     switch (language) {
     case "java":
-      return forBackend(stage, new JavaBackend());
+      return forBackend(stage, new XoomCodeGenBackend(new SchemaTypeTemplateProcessingStep(), language));
```

Comment:
Shall we remove the `Backend` abstraction now that it's unlikely we will have more than one implementation?